mandatory/4-tax.js

```diff
@@ -17,7 +21,13 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  console.log(price)
+  return "£" + calculateSalesTax(price).toFixed(2);
+  
+}
+console.log(addTaxAndFormatCurrency(40))
+
```

Comment:
Consider this option. function addTaxAndFormatCurrency(sales) {
return "£" + calculateSalesTax(sales).toFixed(2);
}